packages/rocketchat-channel-settings-mail-messages/server/methods/mailMessages.js

```diff
@@ -55,7 +55,7 @@ Meteor.methods({
 		if (data.language !== 'en') {
 			const localeFn = Meteor.call('loadLocale', data.language);
 			if (localeFn) {
-				Function(localeFn)();
+				Function(moment.localeFn)();
```

Comment:
Hi, honestly I'm a little clueless with this problem localFn contains the content of private/moment-locales/de.js in case of "de" set in user's profile. What does ```Function(localeFn)();``` do then? For testing I removed the complete code between ```if (data.language !== 'en') {``` and without it, it's still running the the correct locale set to "de".